pharus/dynamic_api_gen.py

```diff
@@ -24,25 +24,26 @@ def populate_api():
 """
     route_template = """
 
-@app.route('{route}', methods=['GET'])
+@app.route('{route}', methods=['GET', 'POST'])
 @protected_route
 def {method_name}(jwt_payload: dict) -> dict:
 
-    if request.method in {{'GET'}}:
+    if request.method in {{'GET', 'POST'}}:
```

Comment:
To fix the tests split the If into an if-elseif for `post` and `get`, in the get route do not do `payload=request.get_json()` just do not set it and set the default to `none` in the `component_interface`.

Looking at the failed tests it seems that `request.get_json()` produces an error when being used in a get route.